pages/api/submissions/index.ts

```diff
@@ -1,31 +1,39 @@
 import forms from 'data/flexibleForms';
 import { NextApiRequest, NextApiResponse } from 'next';
 import StatusCodes from 'http-status-codes';
-import { startSubmission } from 'lib/submissions';
+import { startSubmission, getUnfinishedSubmissions } from 'lib/submissions';
 import { isAuthorised } from 'utils/auth';
 
 const handler = async (
   req: NextApiRequest,
   res: NextApiResponse
 ): Promise<void> => {
+  const user = isAuthorised(req);
   switch (req.method) {
     case 'POST':
       {
         const { formId, socialCareId } = req.body;
-
-        const user = isAuthorised(req);
-
         const submission = await startSubmission(
           formId,
           socialCareId,
           String(user?.email)
         );
-
         res.json(submission);
       }
       break;
     case 'GET':
-      {
+      if (req.query.includeSubmissions) {
```

Comment:
I think this code is fine, there seems to be some issue on staging currently with getting unfinished submissions, should investigate that before merging this in.

Reply:
was to do with api disruption - working fine now